front/.env

```
@@ -1 +1,3 @@
-VITE_API_URL=http://localhost:3000
\ No newline at end of file
+VITE_API_URL=http://localhost:3001
+FEATURE_FLAG_API=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FEATURE_FLAG_API key should go before the VITE_API_URL key

(UnorderedKey)